var/spack/repos/builtin/packages/neovim/package.py

```diff
@@ -122,7 +122,7 @@ class Neovim(CMakePackage):
     with when("@0.6:"):
         depends_on("cmake@3.10:")
         depends_on("gperf@3.1:")
-        depends_on("libiconv@1.15:")
+        depends_on("iconv")
```

Comment:
This one worries me slightly, it loses the minimum version on libiconv if we were to use it...
maybe we could add a conflicts("libiconv@:1.14") or some such?

Reply:
Fixed (I think).

Reply:
Fixed (I think).